docs/log-collector/README.md

```diff
@@ -1,26 +1,57 @@
 # k8s-triliovault Log Collector
-This is a python **pip** module that collects the information mainly yaml configuration and logs from k8s cluster for debugging k8s-triliovault application.
 
-## Requirements
-1. python >= 3.6
+Log collector let you define what you need to log and how to log it by collecting the the logs and events of Pod. Pod Logs can help you understand what is happening inside your application. The logs are particularly useful for debugging problems and monitoring cluster activity, alongside the metadata of all resources related to TrilioVault as either namespaced by providing namespaces name separated by comma or clustered from k8s cluster for debugging k8s-triliovault application. It also collects the CRDs yaml related to TVK and zip them.
```

Comment:
`Log collector let you define what you need to log and how to log it` how? It seems wrong to me. Log collector just collects logs and events of resources.

Reply:
How to log i mean the level of log such as info error warn

Reply:
resolved